python/ray/tests/test_worker_capping.py

```diff
@@ -177,8 +177,8 @@ def f(i, start):
 
     # Assert that last_wwait / second_last ~= 2, with a healthy buffer since ci
     # is noisy.
-    assert 1 < last_wait / second_last < 4
-    assert 2 < last_wait < 4
+    assert second_last < last_wait < 4 * second_last
```

Comment:
Dividing values might result in more noise (might be over optimization of stuffs but since the test is flaky, didn't want it to fail because of slight errors in final values). So directly comparing.